documentation/working-with-testcafe-studio/recording-tests/test-actions/assertions.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/working-with-testcafe-studio/recording-tests/test-actions/assertions.html
+---
+# Assertions
+
+The **Assertions** category contains actions that perform different verifications.
+
+* ![Action icon](../../../../images/actions/assertion-eql-icon.svg) [Equals](#equals)
+* ![Action icon](../../../../images/actions/assertion-not-eql-icon.svg) [Not Equals](#not-equals)
+* ![Action icon](../../../../images/actions/assertion-ok-icon.svg) [Ok](#ok)
+* ![Action icon](../../../../images/actions/assertion-not-ok-icon.svg) [Not Ok](#not-ok)
+* ![Action icon](../../../../images/actions/assertion-contains-icon.svg) [Contains](#contains)
+* ![Action icon](../../../../images/actions/assertion-not-contains-icon.svg) [Not Contains](#not-contains)
+* ![Action icon](../../../../images/actions/assertion-type-of-icon.svg) [Type of](#type-of)
+* ![Action icon](../../../../images/actions/assertion-not-type-of-icon.svg) [Not Type of](#not-type-of)
+* ![Action icon](../../../../images/actions/assertion-gt-icon.svg) [Greater than](#greater-than)
+* ![Action icon](../../../../images/actions/assertion-gte-icon.svg) [Greater than or Equals to](#greater-than-or-equals-to)
+* ![Action icon](../../../../images/actions/assertion-lt-icon.svg) [Less than](#less-than)
+* ![Action icon](../../../../images/actions/assertion-lte-icon.svg) [Less than or Equals to](#less-than-or-equals-to)
+* ![Action icon](../../../../images/actions/assertion-within-icon.svg) [Within](#within)
+* ![Action icon](../../../../images/actions/assertion-not-within-icon.svg) [Not Within](#not-within)
+* ![Action icon](../../../../images/actions/assertion-match-icon.svg) [Match](#match)
+* ![Action icon](../../../../images/actions/assertion-not-match-icon.svg) [Not Match](#not-match)
+
+## Assertion Parameters and Options
+
+When you add an assertion to a test, you need to set up its parameters.
+
+Each assertion has an **Actual** parameter that takes a comparison value. This parameter can take any value. If you need to compare a property of a particular DOM node, you can pass a [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/)'s property to this parameter. You can enter a selector's property to the **Actual** field or generate it during test recording by selecting a target element on the tested webpage.
+
+To generate a Selector's property, follow the steps:
```

Comment:
I'm not sure about `generate a Selector's property` here, maybe better use `choose a Selector's property`